reactjs/src/stores/tenantStore.ts

```diff
@@ -0,0 +1,72 @@
+import { observable, action } from 'mobx';
+import { GetAllTenantOutput } from 'src/services/tenant/dto/getAllTenantOutput';
+import { PagedResultDto } from 'src/services/dto/pagedResultDto';
+import tenantService from 'src/services/tenant/tenantService';
+import CreateTenantInput from 'src/services/tenant/dto/createTenantInput';
+import UpdateTenantInput from 'src/services/tenant/dto/updateTenantInput';
+import { EntityDto } from 'src/services/dto/entityDto';
+import GetTenantOutput from 'src/services/tenant/dto/getTenantOutput';
+
+
+class TenantStore {
+  @observable tenants: PagedResultDto<GetAllTenantOutput>;
+  @observable editTenant: GetTenantOutput;
+
+  @action
+  async create(createTenantInput: CreateTenantInput) {
+  
+    var result = await tenantService.create(createTenantInput);
+    console.log(result);
+    
+    // this.tenants.items.unshift(result);
+   
+    // 
```

Comment:
Remove commented out lines and all `console.log` lines in all JS files.

Reply:
Resolved